### PR TITLE
Add docstrings and remote unused code

### PR DESCRIPTION
**Pull request by accorvin**

### CODE REVIEW

File: cvengine/platform_handlers/base_platform_handler.py [hunk 1]
```diff
@@ -7,19 +7,66 @@
 
 
 class BasePlatformHandler(object):
+    """Base class for running container validations on the target platform
+
+    "Platforms" are the representation in cvengine of the hosts/systems that
+    containers run on, e.g. Docker or Openshift. This base class is used
+    to store common code that is shared by all platforms. Subclasses exist
+    for each target platform. This base class enumerates the variables that
+    should be defined for a given platform, and also containes the functions
```
Comment (johnbieren) [inline cvengine/platform_handlers/base_platform_handler.py]: nit typo contains

File: cvengine/util/run.py [hunk 2]
```diff
@@ -25,6 +45,27 @@ def run_cmd(cmd, virtualenv=None, working_directory=None, env_vars={}):
...
+    """Helper function to run an ansible command
+
+    Execute the specified command in a local shell via ansible. This is used
+    primarily to execute a command on a remote host, using the ssh mechanisms
+    provided by Ansible.
+
+    Args:
+        cmd (str): The command to be executed
+        data (dict): A set of keys containing Ansible connection information
+            includinnd user and ssh_key_path
```
Comment (johnbieren) [inline cvengine/util/run.py]: typo

Comment (accorvin) [reply cvengine/util/run.py]: Good catch. Will fix.

File: cvengine/platform_handlers/base_platform_handler.py (hunk 1 above)
Comment (accorvin) [reply cvengine/platform_handlers/base_platform_handler.py]: Good catch. Will fix.

**Comment by accorvin** [comment]
I'll squash the fixup commit after everyone has had a chance to review.

**Review by robnester-rh — APPROVED** [review]

**Pull request merged**